src/logfire.rs

```diff
@@ -95,6 +95,18 @@ impl Logfire {
         }
     }
 
+    /// Forcibly flush the current data captured by Logfire.
+    ///
+    /// # Errors
+    ///
+    /// This will error if the underlying OpenTelemetry SDK fails to flush data.
+    pub fn force_flush(&self) -> Result<(), opentelemetry_sdk::error::OTelSdkError> {
```

Comment:
Is there an async way to accomplish this?  & is this method blocking & if so maybe we should document it is